packages/apps-sdk/src/sdk.ts

```diff
@@ -1,20 +1,32 @@
 import postRobot from 'post-robot';
+import Listener from './listener';
+
+import { EventData } from './types';
 
 import {
+  EVENT_TYPE_EXECUTE_APP_METHOD,
   EVENT_TYPE_GET_APP_SETTINGS,
   EVENT_TYPE_LOAD_APP,
   EVENT_TYPE_REMOVE_APP,
 } from './constants';
 
-export class SDK {
+export class SDK extends Listener {
```

Comment:
Why SDK extends Listener, maybe it is a context and objective mix? I mean the SDK is not a Listener, perhaps we should has an instance of a listener that SDK will use :dog2:  Because actually you are doing so here:
```javascript
  public registerMethod(method: string, callback: () => void) {
    this.addListener(method, callback);
  }
```